pymavswarm/mavswarm.py

```diff
@@ -1163,15 +1163,20 @@ def executor(agent_id: AgentID) -> None:
                 self._connection.mavlink_connection.target_system = agent_id[0]
                 self._connection.mavlink_connection.target_component = agent_id[1]
 
-                # Send flight mode
+                # Send debug message
                 if isinstance(value, int):
                     self._connection.mavlink_connection.mav.named_value_int_send(
                         int(time.time()), str.encode(name), value
                     )
-                else:
+                elif isinstance(value,float):
                     self._connection.mavlink_connection.mav.named_value_float_send(
                         int(time.time()), str.encode(name), value
                     )
+                elif isinstance(value, list):
```

Comment:
Provided that there is a specific length that the vector should be, I think it would be useful to raise an exception here if the length is not equal to the allowable length (3)